src/NzbDrone.Core/Download/FailedDownloadService.cs

```diff
@@ -83,7 +83,7 @@ public void Check(TrackedDownload trackedDownload)
 
                 if (grabbedItems.Empty())
                 {
-                    trackedDownload.Warn("Download wasn't grabbed by sonarr, skipping");
+                    trackedDownload.Warn("Download wasn't grabbed by Sonarr, skipping automatic download handling");
```

Comment:
should we add in `failed` as well, so someone seeing this doesn't assume any download not grabbed by Sonarr will be skipped since it's just the failed ones?

Reply:
Its not only failed downloads at this warning, also encrypted downloads hit this warning if not grabbed by Sonarr. Can split it out if you want?

Reply:
We do end up treating them both as failed, but handling the messaging differently seems reasonable. To avoid duplicating the whole block, rechecking `trackedDownload.DownloadItem.IsEncrypted` here would probably be easiest.

Reply:
done